src/main/database/mysql-muc-3.4.0.sql

```diff
@@ -16,3 +16,169 @@
 -- If not, see http://www.gnu.org/licenses/.
 --
 
+
+-- QUERY START:
+drop function if exists Tig_MUC_UuidToOrdered;
```

Comment:
Maybe with MUC 3.4 / Tigase 8.4 we should bump minimum required version of mysql to 8 and use native uuid? In 8.3 we considered deprecating 5.7 (https://projects.tigase.net/issue/serverdist-10)

Reply:
I would tend to agree, but I do not want to remove support for MySQL 5.7 in dot-point release. I think it would be good to leave it as it is for 8.4 and replace it with native UUID in Tigase 9.0